Assets/MixedRealityToolkit/Inspectors/Setup/MixedRealityProjectConfiguratorWindow.cs

```diff
@@ -20,12 +20,21 @@ public class MixedRealityProjectConfiguratorWindow : EditorWindow
             {MRConfig.VirtualRealitySupported, true },
             {MRConfig.SinglePassInstancing, true },
             {MRConfig.SpatialAwarenessLayer, true },
+            // UWP Capabilities
             {MRConfig.MicrophoneCapability, true },
             {MRConfig.InternetClientCapability, true },
             {MRConfig.SpatialPerceptionCapability, true },
 #if UNITY_2019_3_OR_NEWER
             {MRConfig.EyeTrackingCapability, true },
 #endif
+            // Android Settings
+            {MRConfig.AndroidMultiThreadedRendering, true },
```

Comment:
We may also want to eventually add a doc page with more explanation of what and why every item is here

Reply:
if you file an issue, i'll write the doc :)